linfa-predictor/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
```

Comment:
Yep working on it here https://github.com/rust-ml/linfa/pull/45 :+1:

Reply:
I have implemented a `LinfaError` struct that is now returned from functions returning `Result` 
Specifically, decision trees should not implement `predict_probabilities()`. In the next commit I gently raise the error instead of the `unimplemented!()`

`Predictor` trait is currently in its own crate. I personally do not think that makes sense (it should be part of linfa, regardless). I have placed it as importable crate for convenience now, but needs to be moved

Reply:
> Specifically, decision trees should not implement `predict_probabilities()`
Do you mean that
1. This is actually genuinely not implemented and we should handle the case more gracefully
2. Making a prediction of class probabilities is not a sensible operation for decision trees?